cmd/cloud-controller-manager/main.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// The external controller manager is responsible for running controller loops that
+// are cloud provider dependent. It uses the API to listen to new events on resources.
+
+package main
+
+import (
+	"k8s.io/cloud-provider-gcp/cmd/cloud-controller-manager/app"
+	"math/rand"
+	"os"
+	"time"
+
+	"k8s.io/component-base/logs"
+	_ "k8s.io/component-base/metrics/prometheus/version" // for version metric registration
+	// NOTE: Importing all in-tree cloud-providers is not required when
+	// implementing an out-of-tree cloud-provider.
+	_ "k8s.io/component-base/metrics/prometheus/clientgo" // load all the prometheus client-go plugins
+	_ "k8s.io/legacy-cloud-providers/gce"
+)
+
+func main() {
+	rand.Seed(time.Now().UnixNano())
+	logs.InitLogs()
+	defer logs.FlushLogs()
+
+	command := app.NewCloudControllerManagerCommand()
+
+	if err := command.Execute(); err != nil {
+		os.Exit(1)
+	}
+	//rand.Seed(time.Now().UnixNano())
```

Comment:
Can we lose this huge chunk of commented out code if we don't need it?

Reply:
Sorry.Updated